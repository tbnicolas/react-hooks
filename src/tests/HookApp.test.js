import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';


describe('Prueba del comoponente <HookApp />', () => {
  

    test('debe de ejecutarse correctamente en el tiempo dicho componente', () => {
        
        const wrapper = shallow( <HookApp /> );

        expect( wrapper ).toMatchSnapshot();

    });
    

});
 