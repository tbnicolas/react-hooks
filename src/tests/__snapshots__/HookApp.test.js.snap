// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba del comoponente <HookApp /> debe de ejecutarse correctamente en el tiempo dicho componente 1`] = `
<div>
  <h1>
     Hello World 
  </h1>
</div>
`;
